### ISSUE TITLE
baseItemWeightReduction has no affect

### ISSUE THREAD

**Issue by dyoung522** [nxPublic/ValheimPlus#12]
```
baseItemWeightReduction=0.8
;(float), removes from value (original - (original * baseItemWeightReduction).
; 1 is 100% weight reduction of every item, 0.5 is 50%.
```
This doesn't seem to have any effect.

**Comment by dyoung522** [comment]
For Reference; I tried the values `0.5` and `0.8` and there was no difference in how much I could carry of a particular resource.

**Comment by nxPublic** [comment]
This issue is resolved in 0.6

**Comment by nxPublic** [comment]
Issue is resolved in 0.6
